xml/System.Text/SpanRuneEnumerator.xml

```diff
@@ -19,8 +19,16 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides an enumerator for the <see cref="T:System.Text.Rune" /> values represented by a span containing UTF-16 text.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+
+This type is not intended to be referenced directly by application code. Instead, use the <xref:System.MemoryExtensions.EnumerateRunes> method along with language-specific enumeration constructs to enumerate <xref:System.Text.Rune> values within spans.
+
+The C# [foreach](~/docs/csharp/language-reference/keywords/foreach-in.md) of the C# language and the [For Each...Next](~/docs/visual-basic/language-reference/statements/for-each-next-statement.md) construct in Visual Basic hides the complexity of enumerators. Instead of directly manipulating the enumerator, using `foreach` or `For Each...Next` is recommended.
```

Comment:
I think I like this version used by https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.enumerator better

```suggestion
The `foreach` statement of the C# language (`for each` in C++, `For Each` in Visual Basic) hides the complexity of enumerators. Therefore, using `foreach` is recommended, instead of directly manipulating the enumerator.
```